cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/workflow/WorkflowProgramController.java

```diff
@@ -95,7 +95,7 @@ public void terminated(Service.State from) {
 
       @Override
       public void failed(Service.State from, Throwable failure) {
-        LOG.info("Workflow service failed from {}. Un-registering service {}.", from, serviceName, failure);
+        LOG.error("Workflow service '{}' failed.", serviceName, failure);
```

Comment:
"service unregistered" is a info message underneath. Also "unregistering service" is not really part of the error.